app/forms/schools/on_boarding/fees.rb

```diff
@@ -4,16 +4,45 @@ class Fees < Step
       attribute :administration_fees, :boolean
       attribute :dbs_fees, :boolean
       attribute :other_fees, :boolean
+      attribute :none, :boolean
+      attribute :selected_fees, default: []
```

Comment:
Initially I thought this model was backed by ActiveRecord and thats why we wanted to use `selected_fees` (so we don't have to change the database) but it looks like that isn't the case; can we not just drop the 4 individual fields and retain `selected_fees`? even if it ends up mapping to individual fields when persisted in the profile I wouldn't have thought that would be too many changes?